rpcs3/Emu/RSX/D3D12/D3D12Formats.cpp

```diff
@@ -382,6 +382,16 @@ BOOL get_front_face_ccw(u32 ffv)
 	throw EXCEPTION("Invalid front face value (0x%x)", ffv);
 }
 
+D3D12_CULL_MODE get_cull_face(u32 cfv) 
+{
+	switch (cfv)
+	{
+	case CELL_GCM_FRONT: return D3D12_CULL_MODE_FRONT;
+	case CELL_GCM_BACK:  return D3D12_CULL_MODE_BACK;
+	}
```

Comment:
Just add a case default and return D3D12_CULL_MODE_NONE instead of trapping with a throw